src/index.js

```diff
@@ -1,29 +1,95 @@
 import './style.css';
 
-const todoListArr = [
-  {
-    description: 'Take the car to the mechanic',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Attend choir practice at 6:00 PM',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'Do some coding challenges on data structures',
-    completed: false,
-    index: 3,
-  },
-];
-
-todoListArr.forEach((todo) => {
-  const todoContent = `
-  <div>
-  <input type="checkbox" class=checkbox>
-  <span>${todo.description}</span>
+const todoListSection = document.getElementById('todo-list');
+const todoText = document.querySelector('input');
+
+class TodoObject {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+const removeTodoList = (todo) => {
+  todoListSection.removeChild(todo);
+  const localdata = JSON.parse(localStorage.getItem('todolist'));
+  const data = Array.from(localdata).filter((i) => i.completed === false);
+  data.map((i) => i.index);
+  localStorage.setItem('todolist', JSON.stringify(data));
+};
+
+const editTodoList = (todoContainer, todoDescription) => {
+  const editInputText = document.createElement('input');
+  editInputText.type = 'text';
+  editInputText.classList = 'editInput';
+  editInputText.value = todoDescription.textContent;
+  todoContainer.replaceChild(editInputText, todoDescription);
+  editInputText.addEventListener('keypress', (e) => {
+    if (e.key === 'Enter') {
+      const todoContainers = document.querySelectorAll('.todoContainer');
+      const storageList = JSON.parse(localStorage.getItem('todolist'));
+      for (let i = 0; i < todoContainers.length; i += 1) {
+        if (todoContainers[i].classList.contains('clickedcheckbox')) {
+          storageList[i].description = editInputText.value;
+          localStorage.setItem('todolist', JSON.stringify(storageList));
+        }
+      }
+      editInputText.parentElement.classList.remove('clickedcheckbox');
+      todoContainer.replaceChild(todoDescription, editInputText);
+      todoDescription.textContent = editInputText.value;
+    }
+  });
+};
+
+const todoListArr = [];
+
+const addTodo = (todoDescription) => {
+  const todoContainer = document.createElement('div');
+  todoContainer.className = 'todoContainer';
+  todoContainer.innerHTML += `
+  <input type="checkbox" class="checkbox">
+  <span>${todoDescription}</span>
   <i class="fas fa-ellipsis-v"></i>
+  <i class="fas fa-trash-alt"></i>
   </div>`;
-  document.getElementById('todo-list').innerHTML += todoContent;
-});
```

Comment:
According to the project requirements, you should have a new JavaScript file for the new functionality.
![image](https://user-images.githubusercontent.com/38088818/173177735-fecece52-2a3b-4dba-977b-8203bd9eaf8c.png)

- Kindly create a new JavaScript file and move all the functions for adding a new task, editing a new task, and deleting a new task to the new JS file and `export` them. You can then `import` each function to the `index.js` file.